clippy_lints/src/attrs/mod.rs

```diff
@@ -160,9 +160,8 @@ declare_clippy_lint! {
     /// are stable now, they should be used instead of the old `cfg_attr(rustfmt)` attributes.
     ///
     /// ### Known problems
-    /// This lint doesn't detect crate level inner attributes, because they get
-    /// processed before the PreExpansionPass lints get executed. See
-    /// [#3123](https://github.com/rust-lang/rust-clippy/pull/3123#issuecomment-422321765)
+    /// `cfg_attr` is valid in some places that `rustfmt::skip` is not, e.g. outer attributes on
+    /// blocks and inner attributes on inline modules.
```

Comment:
Can't we easily not lint here. We know what node the attribute was attached to.

The inner attribute could technically be moved to an outer attribute, but just not linting would be fine.

Reply:
Yeah, they were valid in fewer places than I thought. I *think* I have all the outer ones but may have missed something

I reverted the inner attribute change since a top level `#![rustfmt::skip]` isn't valid, I had just assumed it was based on the comment in the test